helm/prometheus-meta-operator/templates/prometheus-rules/up.all.rules.yml

```diff
@@ -52,7 +52,7 @@ spec:
         cancel_if_cluster_status_creating: "true"
         cancel_if_cluster_status_deleting: "true"
         cancel_if_cluster_status_updating: "true"
-        kube_state_metrics_down: "true"
+        cancel_if_any_kube_state_metrics_down: "true"
```

Comment:
Not totally sure how this is wired up with the old/new setup, but if this inhibition name comes from [here](https://github.com/giantswarm/g8s-prometheus/blob/master/helm/g8s-prometheus/templates/alertmanager/configmap.yaml) then I think the `any` here is incorrect

Reply:
The inhibitions are coming from prometheus-meta-operator now as g8s is gone but cancel_if_any_kube_state_metrics_down does not exist there. We changed it to cancel_if_mc_kube_state_metrics_down but the value is never set  for some reason

Reply:
I am _so confused_

I can only find one instance of the inhibition here:
https://github.com/giantswarm/prometheus-meta-operator/blob/f115e2ddf34959bb39c842c927002dd0d578fbfc/helm/prometheus-meta-operator/templates/prometheus-rules/prometheus-meta-operator.rules.yml#L43

What should the correct value be? I feel like I don't get it at all right now :D

Reply:
Oh wait, I was wrong, we still configure inhibitions in g8s-prometheus ... 
I will see myself out now :D

Reply:
Sorry for your brain @MarcelMue

Reply:
@TheoBrigitte when do we set kube_state_metrics_down to true now?

Reply:
Inhibition are still wired in alertmanager AFAIK, so I added it there.

Reply:
`kube_state_metrics_down` is set by `InhibitionKubeStateMetricsDown`

Reply:
Thanks,  Could you just add a comment here to explain for the future?

Reply:
Feels super weird to have a comment here about this specific label, while we have many other inhibition label all over the place. Skipping comment and moving on.